Automattic/wp-calypso
# Export: Populate advanced settings menus with API data

Populates the Exporter advanced settings dropdown menus with data fetched from the API:

<img width="570" alt="screen shot 2016-02-18 at 6 46 34 pm" src="https://cloud.githubusercontent.com/assets/416133/13138237/a7ac690e-d671-11e5-8163-a641a048da3d.png">

The `OptionFieldset` component has also been renamed `PostTypeOptions` to better describe its purpose.

### Testing

0. ~~Apply patch D1816 and sandbox public-api.wordpress.com~~ Deployed in r135999-wpcom
1. Go to `/settings/export` and choose a site.
2. Click the down chevron beside **Export All**.
3. Confirm that the author, status, date and category options under each post type belong to the site you've chosen.
4. Switch sites.
5. Confirm that the options in the dropdowns have changed and match the new site.

cc @dllh @omarjackman @dmsnell
